backend/src/lib/server-side-events.ts

```diff
@@ -226,12 +227,14 @@ export class ServerSideEvents {
             }
         }
         const eventClient: ServerSideEventClient = {
+            token,
             events,
             namespaces,
             writableStream,
             compressionStream,
             eventQueue: [],
         }
+        const clientID = randomString(8)
```

Comment:
does it mean length with 8? not sure about the possibility of collision, or what about random+token/hash-of-token?

Reply:
I guess 8 should be enough for alphanumeric. it's good enough 👍